src/app/components/form/Form.module.css

```diff
@@ -0,0 +1,41 @@
+.form {
+  display: grid;
+  background-color: rgb(249 233 194);
+  border-radius: 1rem;
+  padding: 2.5rem;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  row-gap: 1rem;
+  column-gap: 0.5rem;
+  align-items: center;
+  justify-items: center;
+}
+
+.form__title {
+  margin: 0;
+  grid-row: 1/2;
+  grid-column: 1/3;
+  color: rgb(35 103 106);
+  text-transform: uppercase;
+  letter-spacing: 0.1rem;
+  font-size: 2rem;
+}
+
+.form__text {
+  grid-row: 2/3;
+  border: hidden;
```

Comment:
I can't seem to find anything for "border: hidden". Did you mean"border: none"?

Reply:
it’s the same as „border-style: hidden“